packages/fx-core/src/plugins/resource/aad/utils/configs.ts

```diff
@@ -8,14 +8,84 @@ import { format, Formats } from "./format";
 import { Utils } from "./common";
 import { ResultFactory } from "../results";
 import { v4 as uuidv4 } from "uuid";
-import { getArmOutput, isArmSupportEnabled } from "../../../../common";
+import { getArmOutput, isArmSupportEnabled, isMultiEnvEnabled } from "../../../../common";
+import {
+  LocalSettingsBotKeys,
+  LocalSettingsFrontendKeys,
+} from "../../../../common/localSettingsConstants";
 
-function checkAndSaveConfig(context: PluginContext, key: string, value: ConfigValue) {
-  if (!value) {
-    return;
+// function checkAndSaveConfig(context: PluginContext, key: string, value: ConfigValue) {
```

Comment:
Why comment out but not remove?

Reply:
Just forgot to remove it . Already clean up.